spec/brn_parser_spec.rb

```diff
@@ -0,0 +1,85 @@
+describe Brn::Parser do
+  before do
+    @brn_parser = Brn::Parser.new
+  end
+
+  describe "integer" do
+    it "single integer" do
+      expect(@brn_parser.integer.parse("1")).to_not be_nil
```

Comment:
Would it be preferable to assert existence with `.to exist` than a double negative here?

Reply:
The resulting values don't respond to `exists?`, so that causes an error/fails the test.